### PR TITLE
Add localisation support for menu tip

### PR DESCRIPTION
**Pull request by ItsShamed**
Noticed while watching recent activity on the `#github-feed` channel on Discord, and thought why this wasn’t a thing already.
This will make the process of adding new tips a bit more tedious though…

### CODE REVIEW

File: osu.Game/Screens/Menu/MenuTip.cs [hunk 1]
```diff
@@ -78,49 +80,49 @@ public void ShowNextTip()
...
+            var tip = getRandomTip();
 
             textFlow.Clear();
-            textFlow.AddParagraph("a tip for you:", formatSemiBold);
+            textFlow.AddParagraph(MenuTipStrings.MenuTipTitle, formatSemiBold);
             textFlow.AddParagraph(tip, formatRegular);
 
             this.FadeInFromZero(200, Easing.OutQuint)
-                .Delay(1000 + 80 * tip.Length)
+                .Delay(1000 + 80 * tip.ToString().Length)
```
Comment (frenzibyte) [inline osu.Game/Screens/Menu/MenuTip.cs:90]: You can do better than that, evaluate the string using `LocalisationManager.GetLocalisedString` and read its length.

**Review by frenzibyte — DISMISSED** [review]

File: osu.Game/Screens/Menu/MenuTip.cs (hunk 1 above)
Comment (peppy) [reply osu.Game/Screens/Menu/MenuTip.cs:90]: if we're going that far, will also need to halve the length if a character is multi-byte unicode.

Comment (frenzibyte) [reply osu.Game/Screens/Menu/MenuTip.cs:90]: .NET seems to handle that well for us, on testing with an Arabic character:

CleanShot 2024-11-25 at 00 22 13

Comment (frenzibyte) [reply osu.Game/Screens/Menu/MenuTip.cs:90]: Sure.

**Review by frenzibyte — DISMISSED** [review]

File: osu.Game/Localisation/MenuTipStrings.cs [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+    {
+        private const string prefix = @"osu.Game.Resources.Localisation.MenuTip";
+
+        /// <summary>
+        /// "Press Ctrl-T anywhere in the game to toggle the toolbar!"
+        /// </summary>
+        public static LocalisableString ToggleToolbarShortcut => new TranslatableString(getKey(@"toggle_toolbar_shortcut"), @"Press Ctrl-T anywhere in the game to toggle the toolbar!");
+
+        /// <summary>
+        /// "Press Ctrl-O anywhere in the game to access options!"
```
Comment (Joehuu) [inline osu.Game/Localisation/MenuTipStrings.cs:18]: Was renamed in lazer.
```suggestion
        /// "Press Ctrl-O anywhere in the game to access settings!"
```

File: osu.Game/Localisation/MenuTipStrings.cs [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+        /// </summary>
+        public static LocalisableString SkinEditor => new TranslatableString(getKey(@"skin_editor"), @"Your gameplay HUD can be customized by using the skin layout editor. Open it at any time via Ctrl-Shift-S!");
+
+        /// <summary>
+        /// "You can create mod presets to make toggling your favorite mod combinations easier!"
+        /// </summary>
+        public static LocalisableString ModPresets => new TranslatableString(getKey(@"mod_presets"), @"You can create mod presets to make toggling your favorite mod combinations easier!");
+
+        /// <summary>
+        /// "Many mods have customisation settings that drastically change how they function. Click the Mod Customisation button in mod select to view settings!"
```
Comment (Joehuu) [inline osu.Game/Localisation/MenuTipStrings.cs:98]:
```suggestion
        /// "Many mods have customisation settings that drastically change how they function. Click the Customise button in mod select to view settings!"
```

File: osu.Game/Localisation/MenuTipStrings.cs [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+        /// </summary>
+        public static LocalisableString ReplayPausing => new TranslatableString(getKey(@"replay_pausing"), @"You can pause during a replay by pressing Space!");
+
+        /// <summary>
+        /// "Most of the hotkeys in the game are configurable and can be changed to anything you want. Check the bindings panel under input settings!"
+        /// </summary>
+        public static LocalisableString ConfigurableHotkeys => new TranslatableString(getKey(@"configurable_hotkeys"), @"Most of the hotkeys in the game are configurable and can be changed to anything you want. Check the bindings panel under input settings!");
+
+        /// <summary>
+        /// "Your gameplay HUD can be customized by using the skin layout editor. Open it at any time via Ctrl-Shift-S!"
```
Comment (Joehuu) [inline osu.Game/Localisation/MenuTipStrings.cs:88]:
```suggestion
        /// "Your gameplay HUD can be customised by using the skin layout editor. Open it at any time via Ctrl-Shift-S!"
```

**Review by Joehuu — CHANGES_REQUESTED** [review]

**Comment by frenzibyte** [comment]
I wouldn't throw a "changes requested" for what are more of corrections based on recent game updates (i.e. irrelevant from this PR) but sure.

**Review by Joehuu — APPROVED** [review]

**Pull request merged**